fmatch/logrus.py

```diff
@@ -0,0 +1,49 @@
+"""
+Logger as a common package
+"""
+
+import logging
+import sys
+
+
+class SingletonLogger:
```

Comment:
I don't think we need a singleton class here. It just as simple as updating a config.

Reply:
Thank you for the suggestion. The thought behind this was as follows: having a logger setup in Matcher class created duplicate logs of the same log-output, especially in daemon mode where multiple Matcher objects would be created as when the request would come in and multiple formatters were being attached to the logger which led to duplicate logs. To mitigate this a temporary solution was to check if the logger existed previously. I transferred this logic to a class so that it can be used in Orion as well. The SingletonLogger here also updates the config using methods and also keeps track of only having a single logger.

Reply:
logging is [singleton by default](https://docs.python.org/3/library/logging.html#logger-objects). All I am saying is we need not write a custom logging class, we can just use the existing ones.